tests/testthat/test-aorsf_data.R

```diff
@@ -0,0 +1,176 @@
+library(testthat)
+
+mtcars_orsf <- mtcars
+mtcars_orsf$vs <- factor(mtcars_orsf$vs)
+mtcars_na <- mtcars_orsf
+mtcars_na$cyl[1] <- NA
+
+
+test_that("regression model object", {
+
+  skip_if_not_installed("aorsf", "0.1.3")
+
+  set.seed(1234)
+  aorsf_regr_fit <- aorsf::orsf(
+    # everyone's favorite
+    data = mtcars_orsf, formula = mpg ~ .,
+    # faster
+    n_tree = 10,
+    # requested default from tidymodels
+    n_thread = 1
+  )
+
+  # formula method
+  regr_spec <- rand_forest(trees = 10) %>%
+    set_engine("aorsf") %>%
+    set_mode("regression")
+
+  # not sure if the expect_error is needed, but copying from censored
+  set.seed(1234)
+  expect_error(
+    bonsai_regr_fit <- fit(regr_spec, data = mtcars_orsf, mpg ~ .),
+    NA
+  )
+
+  expect_equal(
+    bonsai_regr_fit$fit,
+    aorsf_regr_fit,
+    ignore_formula_env = TRUE
+  )
+
+})
+
+test_that("classification model object", {
+
+  skip_if_not_installed("aorsf", "0.1.3")
+
+  set.seed(1234)
+  aorsf_clsf_fit <- aorsf::orsf(
+    data = mtcars_orsf, formula = vs ~ .,
+    n_tree = 10,
+    n_thread = 1
+  )
+
+  # formula method
+  clsf_spec <- rand_forest(trees = 10) %>%
+    set_engine("aorsf") %>%
+    set_mode("classification")
+
+  set.seed(1234)
+  expect_error(
+    bonsai_clsf_fit <- fit(clsf_spec, data = mtcars_orsf, vs ~ .),
+    NA
+  )
+
+  expect_equal(
+    bonsai_clsf_fit$fit,
+    aorsf_clsf_fit,
+    ignore_formula_env = TRUE
+  )
+
+})
+
+test_that("regression predictions", {
+
+  # bug-fix for na_action = 'pass' was pushed in this version
+  skip_if_not_installed("aorsf", "0.1.4.9001")
```

Comment:
I realized I had not implemented `na_action = 'pass'` for classification and regression trees when I wrote this test. Currently, these will only pass with the dev version.